site/content/extend/plugins/community-plugin-marketplace.md

```diff
@@ -0,0 +1,90 @@
+
+1. You open an issue on [The Plugin Marketplace Repository](https://github.com/hanzei/plugin-test/issues/new/choose) using a pre defined template for new plugins. The template contains the checklist above, so you can check the items. Please also points out which commit should be used for the review. You may cut an release candidate (RC) and for it the reviews.
+2. The reviews are requested by the owner. This is done by cloning the following JIRA ticket for every reviewer, assigning it and mentioning the ticket on GitHub. The reviewers point out found issues on the bug tracker of the community plugin. Once all issues are resolved, they comment on the issue stating the positive review and mark the JIRA ticket as resolved.
+    - [PM](https://mattermost.atlassian.net/browse/MM-22224)
+    - [Dev](https://mattermost.atlassian.net/browse/MM-22221)
+    - [Security](https://mattermost.atlassian.net/browse/MM-22225)
+    - [QA](https://mattermost.atlassian.net/browse/MM-22223)
+3. Once all items are checked and reviews made, the community repository is forked under the Mattermost GitHub organization as a private repository. This allows using the existing build tools for releasing new plugin versions. The fork is maintained by the owner.
+4. `/mb cutPlugin --repo $REP --tag $TAG` is run to build, signs and uploads the approved commit of the plugin.
+5. The owner opens a PR, which adds the plugin to `plugins.json` using `generator add $REP $TAG --community`. Only a functional review by one dev and one QA member is needed for this PR.
+6. After the PR is merged, the plugin gets promoted across Mattermost social media and swag sent to the maintainer. If there are multiple maintainers, everyone gets swag.
+
+### Process for updating community plugin to the Marketplace
+The process is the same as for adding a community plugin besides a couple of differences:
+- The following JIRA ticket is used instead
+    - [PM](https://mattermost.atlassian.net/browse/MM-22228)
+    - [Dev](https://mattermost.atlassian.net/browse/MM-22222)
+    - [Security](https://mattermost.atlassian.net/browse/MM-22226)
+    - [QA](https://mattermost.atlassian.net/browse/MM-22227)
+- Swag is not sent again
+- Promotion via social media happens only on outstanding updates.
+- The master branch of the fork synced with the upstream community repository by running:
+
+```sh
+git fetch upstream
+git checkout master
+git --ff-only upstream/master
+```
+
+### Beta plugins
+If a community plugin doesn’t not make it through the review process, it may still be added to the Marketplace and market as “Beta” via a label. The reviewers make a case by case decision if the quality of a plugin is sophisticated enough to be added to the Marketplace. Security and functional review and Item 1, 2, 3 and 5 from the [Product Requirements Checklist](#requirements-for-adding-community-plugin-to-the-marketplace) must be fulfilled for beta plugin.
```

Comment:
```suggestion
If a community plugin doesn’t make it through the review process, it may still be added to the Marketplace and marked as “Beta” via a label. The reviewers make a case by case decision if the quality of a plugin is sophisticated enough to be added to the Marketplace. Security and functional reviews and Items 1, 2, 3 and 5 from the [Product Requirements Checklist](#requirements-for-adding-community-plugin-to-the-marketplace) must be fulfilled for beta plugins.
```